db/migrate/20221005163229_update_okc_blotter_pdfs.rb

```diff
@@ -0,0 +1,5 @@
+class UpdateOkcBlotterPdfs < ActiveRecord::Migration[6.0]
+  def change
+    change_column(:okc_blotter_pdfs, :parsed_on, :datetime)
```

Comment:
we try to merge migration files that act on the same table into one but these migrations are a bit finicky so doing as two

Reply:
Understandable